script.js

```diff
@@ -81,6 +83,16 @@ function checkWin() {
   return false;
 }
 
+function clearGameBoard(event) {
+  if (text.innerHTML === "PLAYER " + xGamePiece + " WINS!") {
+    text.innerHTML = "Player X turn";
+  }
+  if (text.innerHTML === "PLAYER " + oGamePiece + " WINS!") {
+    text.innerHTML = "Player O turn";
+  }
+  gameSpaces.forEach((space) => (space.innerHTML = ""));
```

Comment:
so currently with this logic your clearGameboard function will clear the board even if the game isn't finished yet. 
Is this the desired behavior?